ir-engine/etherealengine-archive
# IR-2721: Zendesk integration

## Summary

Zendesk widget integration

## Subtasks Checklist

## Breaking Changes

## References
closes #_insert number here_

## QA Steps

- Set up a Zendesk key in your environment file
- Go to the console
- it shouldn't be displayed when you are not authenticated
- Log in
- It should display a button in the lower right corner which opens a messaging window.  
- This functionality is only for authenticated users
